python/mxnet/executor.py

```diff
@@ -140,6 +148,14 @@ def backward(self, out_grads=None):
             Gradient on the outputs to be propagated back.
             This parameter is only needed when bind is called
             on outputs that are not a loss function.
+
+        Examples
+        --------
+        >>> # executor is created by binding on loss function
```

Comment:
make two examples.
explain what's loss function and what's not. explain what out_grads are used for